Source/LocalNetAppChat/LocalNetAppChat.Bot/LocalNetAppChat.Bot.csproj

```diff
@@ -7,6 +7,10 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Cake.Powershell" Version="2.0.0" />
```

Comment:
Please do not use an external library for this. Executing a process and just grabbing the output is a very general process that does not have the need for a dependency on someone else. Especially since we do not know how they bind powershell.

Reply:
Good Point! Will do 👍